doc/builtin_entities.txt

```diff
@@ -0,0 +1,96 @@
+# Builtin Entities
+Minetest registers two entities by default: Falling nodes and dropped items.
+This document describes how they behave and what you can do with them.
+
+## Falling node (`__builtin:falling_node`)
+
+Is created when a node with the special group `falling_node=1` starts to fall.
+Falling nodes can also be created artificially with
+`minetest.spawn_falling_node`.
+
+Spawning a falling node with `/spawnentity` won't work. 
+
+Behaviour:
```

Comment:
`Default behaviour:`

This would also indicate that mods can manipulate this entity. For example: altered horizontal/vertical speed.